.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -41,3 +67,11 @@ jobs:
           cp -v example_data/* data/
 
       - run: nextstrain build --docker . ${{ inputs.build-args }}
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: outputs
+          path: |
+            auspice/
+            results/
+            .snakemake/log/
```

Comment:
I noticed for ncov workflow there are also `benchmarks/` and `logs/` directories generated, is that specific to ncov workflow or not useful here?

Reply:
Their presence is workflow-dependent (more so than `auspice/` and `results/`) but the names are conventional when we use them. I think it's fine to include patterns here that don't always match; when I typo'd `.snakemake/logs/` (note the plural) nothing broke it just didn't upload the log files because nothing matched. That said, I'm not sure that `benchmarks/` is all that useful for CI runs, where I expect we're always using vastly-reduced example datasets with runtime characteristics that aren't representative. `logs/` is probably useful though, if it exists!

Reply:
Added both in 3cc214b.